mpop_service/main.cpp

```diff
@@ -8,19 +8,20 @@
 #include <QProcessEnvironment>
 #include "mpopservice.h"
 #include "config.h"
+#include "logutils.h"
 #include <QDebug>
 
 
 int main(int argc, char *argv[])
 {
     QCoreApplication application(argc, argv);
     application.setApplicationName("mpop_service");
+    logutils();
```

Comment:
We should probably store that object in a variable, like this: `LogUtils = LogUtils();`